docs/devguide/flashfirmware.rst

```diff
@@ -1,16 +1,32 @@
 .. _flashfirmware:
 
-=================
-Flashing Firmware
-=================
+========================
+Build and flash firmware
```

Comment:
Let's rename this (and the link on the left menu) to say `Build and flash firmware`.
We've used the word "firmware" very early on the MicroPython nomenclature, but it can be used with the use of "firmware" in the rest of the micro:bit documentation referring to the DAPLink firmware.

Reply:
@microbit-mark sorry, I meant `Build and flash MicroPython` here, as I wanted to use the word firmware too much.

Reply:
Done in https://github.com/bbcmicrobit/micropython/pull/711/commits/939efde1f837cd9aea60e972e3bb135b215baff6